chrome/content/zotero/elements/itemPaneSection.js

```diff
@@ -126,12 +126,11 @@ class ItemPaneSectionElementBase extends XULElementBase {
 		let key = `_${type}RenderItemID`;
```

Comment:
If we are setting an array to this property, maybe better to use a different name? Renaming it may also affect some tests/components.